apps/speed.c

```diff
@@ -2122,6 +2205,20 @@ int speed_main(int argc, char **argv)
     }
 #  endif
 
+#  ifndef OPENSSL_NO_DH
+    ffdh_c[R_FFDH_2048][0] = count / 1000;
+    for (i = R_FFDH_3072; i <= R_FFDH_8192; i++) {
+        ffdh_c[i][0] = ffdh_c[i - 1][0] / 2;
+        if (ffdh_doit[i] <= 1 && ffdh_c[i][0] == 0)
+            ffdh_doit[i] = 0;
+        else {
+            if (ffdh_c[i][0] == 0) {
+                ffdh_c[i][0] = 1;
+            }
```

Comment:
No need to use {} here because its a single line "if"

Reply:
fixed